src/main/groovy/com/excelsiorjet/gradle/plugin/ExcelsiorJetPlugin.groovy

```diff
@@ -58,34 +58,36 @@ class ExcelsiorJetPlugin implements Plugin<Project> {
 
         extension = target.getExtensions().create(ExcelsiorJetExtension.EXTENSION_NAME, ExcelsiorJetExtension)
 
-        def archiveTaskName
+        Task archiveTask
```

Comment:
It's more efficient. We don't really need archiveTaskName, it is only used for getting the real task.